src/cli/index.js

```diff
@@ -1,189 +1,70 @@
-import fs from 'fs';
-import chalk from 'chalk';
+import path from 'path';
+import { spawn } from 'child_process';
+import semver from 'semver';
 import resolveCwd from 'resolve-cwd';
-import browserProviderPool from '../browser/provider/pool';
-import { GeneralError, APIError } from '../errors/runtime';
-import MESSAGE from '../errors/runtime/message';
-import CliArgumentParser from './argument-parser';
 import log from './log';
-import remotesWizard from './remotes-wizard';
-import createTestCafe from '../';
+import TerminationHandler from './termination-handler';
 
 
-const TERMINATION_TYPES = {
-    sigint:   'sigint',
-    sigbreak: 'sigbreak',
-    shutdown: 'shutdown'
-};
+const SAFE_VERSION = '0.19.0-alpha2';
```

Comment:
`SAFE_VERSION` is not clear name.

Something like
`const MINIMAL_VERSION_WITH_SINGLE_STACK_CHAIN_MODULE_IN_PROCESS =...`
Also, add comment why we do this.